tests/utils/tests.nim

```diff
@@ -0,0 +1,29 @@
+import testutils/unittests
+
+template xsuite*(name: string, body: untyped) =
+  discard
+
+template suitex*(name: string, body: untyped) =
+  discard
+
+template xprocSuite*(name: string, body: untyped) =
+  discard
+
+template procSuitex*(name: string, body: untyped) =
+  discard
+
+template xtest*(name: string, body: untyped) =
+  test name:
+    skip()
+
+template testx*(name: string, body: untyped) =
+  test name:
+    skip()
+
+template xasyncTest*(name: string, body: untyped) =
+  test name:
+    skip()
+
+template asyncTestx*(name: string, body: untyped) =
```

Comment:
This isn't necessary, if a test should not run, just remove it.

Reply:
The usage of `x` to specify skipped test is not my invention. This module has been sourced from Waku, and it's also used in some other testing libraries in other languages. An example of this would be pytest, IIRC.

There's a lot of reasons why you would want tests to not be run. Specifically, I used it to skip a test which is required, but can't be enabled due to not being able to fully assert the case.

Reply:
That's not a pattern in the libp2p codebase, please remove it.

Reply:
Also, we shouldn't have commented or any code that's unnecessary/not being used.

Reply:
Well, ideally we would enable a way for that to be tested as that would provide better coverage. If you don't want to do that then I'll happily remove it.

Reply:
I would suggest that you create an issue and it can get prioritized. The test can be posted there.